js_modules/dagster-ui/packages/ui-core/src/assets/AssetLink.tsx

```diff
@@ -44,3 +34,17 @@ export const AssetLink = (props: {
     </Box>
   );
 };
+
+export const buildAssetPathString = (path: string[], isGroup?: boolean) => {
+  return (
+    path
+      .reduce((accum, elem, ii) => {
+        if (ii > 0) {
+          accum.push(' / ');
+        }
+        accum.push(elem);
+        return accum;
+      }, [] as string[])
```

Comment:
Would `path.join(' / ')` work? We also have `displayNameForAssetKey` which might do this now,

I think this code was very very old :-)